apps/foundation-demo/src/app/cart/components/minicart-item/minicart-item.component.spec.ts

```diff
@@ -49,47 +49,47 @@ describe('CheckoutCartItemComponent', () => {
     mockCartItem = cartItemFactory.create({image: productImageFactory.create()});
 
     wrapper.cartItemValue = mockCartItem;
-    checkoutCartItemComponent = fixture.debugElement.query(By.css('demo-checkout-cart-item')).componentInstance;
+    cartItemComponent = fixture.debugElement.query(By.css('demo-minicart-item')).componentInstance;
 
     fixture.detectChanges();
   });
 
   describe('when cart has a cartItem', () => {
```

Comment:
This describe should be removed. It doesn't matter if cart has a cartItem or not